cache.js

```diff
@@ -0,0 +1,8 @@
+const redis = require('redis');
+const client = redis.createClient();
+
+client.on("error", function(err) {
+    console.log("Error " + err);
```

Comment:
If there's an error, it is output. We could recover the connection so that we wouldn't lose the caching feature.